.mergify.yml

```diff
@@ -11,9 +11,9 @@ pull_request_rules:
         message: |
           Thanks for submitting this pull request.
           Bors-ng will now build test images. When it succeeds, we will continue to review and test your PR.
-          
+
           bors try
-          
```

Comment:
These indentations are required to continue the multiline string. Please use space characters for the indendation.

Reply:
Argh, sorry for not having paid attention. Thanks for the pointer!